src/layout/footer/index.module.scss

```diff
@@ -0,0 +1,42 @@
+.footer {
+    background-color: #eee;
```

Comment:
use sass variables pls `$white-text`

Reply:
why white-text? this is background, not text.

Reply:
That's right, that name does not make sense if it should be used as background. Anyway, do we need to add `background-color` for that footer? Couldn't we use `Section` component to color that section respectively?